docs/src/main/asciidoc/security-jpa-concept.adoc

```diff
@@ -0,0 +1,180 @@
+[id="security-jpa-concept"]
+= Quarkus Security with JPA
+include::_attributes.adoc[]
+:categories: security
+
+JPA is an identity provider, similar to xref:security-jdbc.adoc[JDBC], suitable for use with the xref:security-basic-authentication-concept.adoc[Basic] and xref:security-authentication-mechanisms-concept.adoc#form-auth[Form-based] Quarkus Security mechanisms, which require a combination of username and password credentials.
+
+The JPA `IdentityProvider` creates a `SecurityIdentity` instance, which is used during user authentication to verify and authorize access requests making your Quarkus application secure.
+
+For an example of practical use of Basic authentication and JPA, see the xref:security-basic-authentication-tutorial.adoc[Secure a Quarkus application with Basic authentication and JPA] tutorial.
+
+
+== JPA entity specification
+
+Quarkus security offers a JPA integration to collect usernames, passwords, and roles, and store them into JPA database entities.
+
+The following JPA entity specification demonstrates how users' information needs to be stored in a JPA entity and properly mapped so that Quarkus can retrieve this information from a database.
+
+
+* The `@UserDefinition` annotation must be present on a JPA entity, regardless of whether link:https://quarkus.io/guides/hibernate-orm-panache[simplified Hibernate ORM with Panache] is used or not.
```

Comment:
@MichalMaler - Sorry just noticed this now while verifying single-sourced content on downstream docs. Should this be an xref? Thinking if not an xref, then later it might present a version mismatch as the link always points to the latest.